### ISSUE TITLE
精度无法复现

### ISSUE THREAD

**Issue by gumowangfei** [LeePleased/StackPropagation-SLU#4]
ATIS 数据集上：  训练参数  python train.py -wed 256 -ehd 256 -aod 128 

Accepted performance: (0.95540278512251, 0.9664053751399776, 0.8622620380739082) at test dataset;

**Comment by LeePleased** [comment]
Thanks for your interest in our work!
We think different environment can cause different results.
We kindly suggest that you can tune the hyper-parameters (dropout, hidden-size, embedding size)  and we believe you can find the better results.
